.circleci/config.yml

```diff
@@ -231,6 +231,14 @@ workflows:
           requires:
             - x230-hotp-maximized
 
+      # Coreboot Nitropads
+      - build_and_persist:
+          name: nitropad-nv41
+          target: nitropad-nv41
+          subcommand: ""
+          requires:
+            - prep_env
```

Comment:
Please depend on librem_14 instead. 

Logic here is that we try to only build musl-cross and tools built with musl-cross once and pass workspaces to other boards sharing the most things in common for the same architecture (x86). 

That way, we don't use CircleCI to rebuild musl-cross-make nor modules that were already built (cache is used to rebuild any changes if needed) and only builds coreboot and linux.


By depending on librem_14, you will share the same cache for linux with librems, which now share the same linux kernel version.

Reply:
@daringer I'm testing your config. For some reason, your fork is not built by circleci. I pushed exact for build test at https://app.circleci.com/pipelines/github/tlaurion/heads/1831/workflows/eab77bd8-f556-4927-ab4d-18e2488f8fd6

Hold on, you might have done the right thing. If save cache works, your approach might speed the builds, but I doubt save_cache can merge multiple directories that are shared across archoitectures (it should fail, we will see). Directories need to be unique, which is why save_cache only had one x86 and one ppc64 cache to merge.

Reply:
updated based on https://github.com/osresearch/heads/pull/1417#pullrequestreview-1544417462